### ISSUE TITLE
Use processing parameters in proc/procs to reprocess FIDs for Bruker NMR data

### ISSUE THREAD

**Issue by tilfischer** [cheminfo/nmrium#2830]
Dear all,

**Is your enhancement request related to a problem? Please describe.**

NMRium actually already accepts Bruker data only consisting of the fid and acqus file. For some types of data such as 1H NMR automated processing takes place and the processed spectrum is displayed. Nice feature! Thank you for that! If not automated processing takes place, the FID is shown and users may reprocess the data. 

In IconNMR one might also decide to save Bruker data only having the processing parameters in expno but not the processed spectra (1r/1i, 1D case). In such a case, NMRium does not seem to eat the proc/procs in expno but starts its automated processing with its own parameters or displays the FID, if automated processing for this type of data is not implemented in NMRium yet. 

I am also aware of operators using TopSpin, doing manual processing and using and script after saving the data to remove the processed spectra again, while keeping the processing parameters proc/procs. Possibly, this is due to the fact that they want to save disk space (no joke) and that they were used not to have processed spectra in their dataset as they used Varian instruments for decades. I must actually also agree that this is the more elegant solution, as the processed spectra could be reobtained from the FID and processing parameters anyhow. 

Overall, I currently see three different types of (current) Bruker data: 1) No pdata/procno (see first paragraph) 2) Only processing parameters in procno. 3) Processing parameters and processing spectra in procno.

For 1) users may reprocess by themself or NMRium offers automated processing. For 3) NMRium might read the 1r/1i (1D case) as it is already the case but for 2) NMRium should actually read the processing parameters to reprocess the spectra. In Bruker TopSpin, this is done with ``trf``.

Moreover, there would be less demand for automated processing of NMRium would reprocess the FID using the processing parameters, if not processed spectra are available, case ``2)`` + Varian, see https://github.com/cheminfo/nmrium/issues/2829. 

**Describe the solution you'd like**

NMRium should eat the processing parameter file proc/proc to reprocess the FID and display the processed spectrum to the users. 

**Describe alternatives you've considered**

NMRium could also ask the user whether to use the processing parameters uploaded or do automated processing based on own parameters, as this is already the case for 1H data. 

Best,
Tillmann

**Comment by lpatiny** [comment]
I agree that the processed data are often removed because they are also bigger that the original data and it would make sense to read those parameters. We however currently don't have all the processing features that exists in topspin but we could try to use the one we have. As far as I know PH0 and PH1 are already extracted from the proc file.

**Comment by lpatiny** [comment]
Varian 1D spectra with processing parameters
[Archive.zip](https://github.com/cheminfo/nmrium/files/14210829/Archive.zip)

First step would be to parse the Varian proc file. @tilfischer would you have something the description of the file format ?

**Comment by tilfischer** [comment]
Dear all,

I will post a link for test data of type ``2)`` soon. I still wait for people to send test data including 1H, 13C, COSY, NOESY, HSQC, HMBC and possible APT or DEPT.

Best,
Tillmann

**Comment by tilfischer** [comment]
Dear all,

please excuse the slight delay. Here is some test data: 
[NMRrium_Mes-Acr-BF4.zip](https://github.com/cheminfo/nmrium/files/14420301/NMRrium_Mes-Acr-BF4.zip)

The data contains data from 1H, 13C, APT, HSQC, HMBC, COSY, NOESY, 19F, 11B experiments. Attribution to [Jannik Selka and Kirsten Zeitler](https://www.chemie.uni-leipzig.de/en/institute-of-organic-chemistry/zeitler-research-group) for providing that data. 

I had a quick test at nmrium.org and the 1H, 13C, APT, 19F and 11B were automatically processed by NMRium, while the phase might need another manual correction. 

I hope this is useful to make this data fly in NMRium.

Best,
Tillmann
